consensus/snapshot/BUILD

```diff
@@ -1,15 +1,37 @@
+filegroup(
+    name = "snapshot_files",
+    srcs = [
+        "@snapshot_conf_mainnet//file",
+        "@snapshot_conf_testnet//file",
+        "@snapshot_mainnet//file",
+        "@snapshot_sig_mainnet//file",
+        "@snapshot_testnet//file",
+    ],
+    visibility = ["//visibility:public"],
+)
+
 cc_library(
     name = "snapshot",
     srcs = ["snapshot.c"],
     hdrs = ["snapshot.h"],
+    data = [":snapshot_files"],
+    defines = [
+        "SNAPSHOT_CONF_MAINNET='\"external/snapshot_conf_mainnet/file/snapshot.json\"'",
```

Comment:
same config_setting / select thing here, see how trit_encoding is setup for reference

Reply:
But unlike trit_encoding, mainnet/testnet is a runtime option, not a compile-time option so we need to have both available ?

Reply:
As discussed, this will be handled in another PR where the network type would be a compile-time option.